### PR TITLE
Fix fetch option in i3s-tile-converter

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

File: modules/tile-converter/src/i3s-converter/i3s-converter.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ export default class I3SConverter {
 
     const fetchOptions = await this._fetchPreloadOptions();
     const sourceTilesetJson = await load(inputUrl, CesiumIonLoader, {
-      fetch: fetchOptions
+      fetch: {fetch: {headers: fetchOptions.headers}, throws: true}
```
Comment (ibgreen) [inline modules/tile-converter/src/i3s-converter/i3s-converter.js:91]: This is too confusing. Let's land #1549 first.

Comment (dryabinin-actionengine) [reply modules/tile-converter/src/i3s-converter/i3s-converter.js]: I updated PR after your changes [here](https://github.com/visgl/loaders.gl/pull/1549)

File: modules/tile-converter/src/i3s-converter/i3s-converter.ts [hunk 2]
```diff
@@ -110,7 +110,8 @@ export default class I3SConverter {
 
     const fetchOptions = await this._fetchPreloadOptions();
     const sourceTilesetJson = await load(inputUrl, CesiumIonLoader, {
-      fetch: fetchOptions
+      fetch: {headers: fetchOptions.headers},
```
Comment (ibgreen) [inline modules/tile-converter/src/i3s-converter/i3s-converter.ts:113]: Is it your intention to discard all other fetchOptions and just pass in headers?
```suggestion
      fetch: fetchOptions,
```

File: modules/tile-converter/src/i3s-converter/i3s-converter.ts [hunk 3]
```diff
@@ -1089,12 +1090,13 @@ export default class I3SConverter {
...
     this.sourceTileset.options = {...this.sourceTileset.options, ...options};
+
     if (options.headers) {
-      this.sourceTileset.fetchOptions.headers = options.headers;
+      this.sourceTileset.loadOptions.fetch.headers = options.headers;
       console.log('Authorization Bearer token has been updated'); // eslint-disable-line no-undef, no-console
     }
     if (options.token) {
-      this.sourceTileset.fetchOptions.token = options.token;
+      this.sourceTileset.loadOptions.fetch.token = options.token;
```
Comment (ibgreen) [inline modules/tile-converter/src/i3s-converter/i3s-converter.ts:1099]: loadOptions.fetch should be a [`RequestInit`]*https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch) object accepted by fetch. 

I don't think is has `token`.

**Review by ibgreen — APPROVED** [review]

**Comment by dryabinin-actionengine** [comment]
Closed because of [duplication PR](https://github.com/visgl/loaders.gl/pull/1571)

**Pull request closed without merging**